site/src/i18n/i18n.ts

```diff
@@ -0,0 +1,20 @@
+import i18next from "i18next"
+import { initReactI18next } from "react-i18next"
+import { en } from "./en"
+
+export const defaultNS = "common"
+export const resources = { en } as const
+
+export const i18n = i18next.use(initReactI18next)
+
+i18n
+  .init({
+    fallbackLng: "en",
+    interpolation: {
+      escapeValue: false, // not needed for react as it escapes by default
+    },
+    resources,
+  })
+  .catch((error) => {
+    console.error(error)
```

Comment:
❓ What will `error` look like? Would prefixing the error messages with something like `[Translation Service]` help us distinguish these vs. others?

Reply:
I will add that! But really, I added this catch block to get around a [linting](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-floating-promises.md) error: `no-floating-promises`
It's not really adding a lot beside quieting lint AFAIK.

Reply:
Oh we are going to add that to code-server! I think it can be super helpful in insuring you don't accidentally forget to `await` a promise. Example: ideally it would catch issues like [this one](https://github.com/coder/code-server/pull/5442/files)